src/text-distinction/text-distinction.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { View, Text, TextInput } from 'react-native';
+
+export default class StringDistinction extends Component {
```

Comment:
we could use an functional component for this.

Reply:
what about PureComponent?

Reply:
@suwu150 the component has no state so a function component will serve well